### DISCUSSION TITLE
Denúncia de uso da plataforma para fins ilícitos [ Reporting use of the platform for illegal purposes ]

### DISCUSSION THREAD

**Discussion by antonio-abrantes** [vercel/community#6073] [Help] [answered]
## Atenção
   A pouco me deparei com um projeto hospedado na plataforma, e este é usado para fins ilícitos, para a coleta de dados e cobrança de valores a pessoas inocentes/desavisadas. Com a promessa de receberem **'dinheiro esquecido'** pago pelo governo, o site pede dados das pessoas e depois gera uma cobrança que supostamente é uma porcentagem do valor que ela receberia, a qual ela tem que pagar para que o valor seja 'liberado', e ao pagar, o valor do montante informado, vai ser enviado para a conta, PIX, ou que quer que ela tenha informado, o que não acontece, e se a pessoa pagar, já era!

   Não sei se cabe aqui esta denúncia, mas seria interessante se mais usúarios que vierem a descobrir outros 'projetos' como este, denunciem, pois isso mancha a imagem da plataforma e da nossa comunidade, e acredito que a maioria de nós, devs, temos uma facilidade maior que a grande maioria das pessoas, de identificar estes casos. Vamos juntos, **expurgar** esses tipos daqui.

**Comment by amyegan** [comment, marked as answer]
Thank you for reporting this @antonio-abrantes. Please send this report to [abuse@vercel.com](mailto:abuse@vercel.com) so it can be properly tracked in our system. I'll let the team know to be on the lookout for your email :bow:

**Reply by amyegan** [reply]
The team reviewed and disabled the site. Thanks again for reporting it!
